.github/workflows/check-repos.yml

```
@@ -8,43 +8,34 @@ on:
   workflow_dispatch:
 
 jobs:
-  checking-repos:
+  ci:
+    uses: ./.github/workflows/run-code-quality-and-tests-no-cache.yml@master
```

actionlint (1.7.12)

[error] could not read reusable workflow file for "./.github/workflows/run-code-quality-and-tests-no-cache.yml@master": open /home/jailuser/git/.github/workflows/run-code-quality-and-tests-no-cache.yml@master: no such file or directory

(expression)